openssl/src/symm.rs

```diff
@@ -22,6 +22,45 @@
 //!       \xFB\x3C\x5E\xC4\x59\x72\x4A\xF4\x7C\xA1",
 //!     &ciphertext[..]);
 //! ```
+//!
+//! Encrypting an assymetric key with a symmetric cipher
+//!
+//! ```
+//! use openssl::rsa::{Padding, Rsa};
+//! use openssl::symm::Cipher;
+//! use std::fs::{File, remove_file};
+//! use std::io::prelude::{Read, Write};
+//!
+//! {   // Generate, encrypt and store keypair:
+//!     let keypair = Rsa::generate(2048).unwrap();
+//!     let cipher = Cipher::aes_256_cbc();
+//!     let pubkey_pem = keypair.public_key_to_pem_pkcs1().unwrap();
+//!     let privkey_pem = keypair.private_key_to_pem_passphrase(cipher, b"Rust").unwrap();
+//!     File::create("localhost.pub").unwrap().write_all(&pubkey_pem).unwrap();
+//!     File::create("localhost.priv").unwrap().write_all(&privkey_pem).unwrap();
+//! }
+//!
+//! // Load private and public key from generated file:
```

Comment:
Why are we writing the keys to disk and then immediately reading them again?

Reply:
The intention was just to show an easy to understand and usable example. It's easy to split into 2 functions. If a reader needs to generate and store an encrypted key, they can use the first part, if they need to load keys from disk, they can use the second part.

I can split it into 2 examples, or shorten it down by removing all File I/O, if you prefer?

Reply:
Yeah, let's split them up and avoid the File IO.